dev/Repeater/ItemsRepeater.h

```diff
@@ -10,12 +10,14 @@
 #include "ItemsRepeaterElementClearingEventArgs.h"
 #include "ItemsRepeaterElementIndexChangedEventArgs.h"
 #include "ItemsRepeater.g.h"
+#include "ItemsRepeater.properties.h"
 #include "ViewportManager.h"
 
 class VirtualizationInfo;
 
 class ItemsRepeater :
-    public ReferenceTracker<ItemsRepeater, DeriveFromPanelHelper_base, winrt::ItemsRepeater, winrt::IItemsRepeater2>
+    public ReferenceTracker<ItemsRepeater, DeriveFromPanelHelper_base, winrt::ItemsRepeater, winrt::IItemsRepeater2>,
```

Comment:
>IItemsRepeater2 [](start = 100, length = 15)

curious when we got ItemsRepeater2

Reply:
It’s from this slicing of the runtimeclass: 
```idl
[WUXC_VERSION_PREVIEW]
{
    ElementAnimator Animator{ get; set; };
}
```

Reply:
ok. that makes sense.